.github/workflows/reusable_fuzzer.yml

```diff
@@ -0,0 +1,66 @@
+on:
+  workflow_call:
+    inputs:
+      fuzzer: 
+        required: true
+        type: string
+      data:
+        required: true
+        type: string
+      timeout-minutes:
+        required: true
+        type: number
+        default: 120
+      dry:
+        required: true
+        type: string
+      max_queries:
+        required: true
+        type: number
+        default: 1000
+    secrets:
+      FUZZEROFDUCKSKEY:
+        required: false
+      DUCKDB_HASH: 
+        required: false
+
+jobs:
+  fuzzer:
+    name: Fuzzer
+    runs-on: ubuntu-latest
+    timeout-minutes: ${{ inputs.timeout-minutes }}
+    steps:
+      - name: checkout duckdb_sqlsmith
+        uses: actions/checkout@v4
+        with:
+          repository: hmeriann/duckdb_sqlsmith
+          path: duckdb_sqlsmith
+          sparse-checkout: |
+            scripts
+
+      - name: Download a single artifact
+        uses: actions/download-artifact@v4
+        with:
+          name: duckdb
+
+      - name: Fuzz
+        shell: bash
+        run: |
+          chmod +x duckdb
+          runtime="1 minute"
+          endtime=$(date -ud "$runtime" +%s)
+          cd duckdb_sqlsmith
+      
+          while [[ $(date -u +%s) -le $endtime ]]
+          do
+              echo "Time Now: `date +%H:%M:%S`"
+              python3 scripts/run_fuzzer.py --${{ inputs.dry }} --no_checks --${{ inputs.fuzzer }} --${{ inputs.data }} --max_queries=${{ inputs.max_queries }} --shell=../duckdb
+          done
+
+      - name: Upload fuzz_clogs
+        if: always()
+        uses: actions/upload-artifact@v4
+        with:
+          name: fuzz_complete_logs_${{ inputs.fuzzer }}_${{ inputs.data }}.sql
+          path: duckdb_sqlsmith/sqlsmith.complete.log
+          compression-level: 9
```

Comment:
Did you look into what the compression level does exactly? If there isn't much of a difference you can just leave it out

Reply:
Github documentation says that the higher compression level is good for the text files, but it could slow down execution a bit. So it depends on the priorities. I assume we need it to run faster, so I'll remove `compression-level`.